orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/jedis/JedisExecutionRepository.groovy

```diff
@@ -583,6 +583,8 @@ class JedisExecutionRepository implements ExecutionRepository {
     map["stage.${stage.id}.parentStageId".toString()] = stage.parentStageId
     map["stage.${stage.id}.requisiteStageRefIds".toString()] = stage.requisiteStageRefIds?.join(",")
     map["stage.${stage.id}.scheduledTime".toString()] = String.valueOf(stage.scheduledTime)
+    map["stage.${stage.id}.timeoutMs".toString()] = stage.timeoutMs?.toString()
```

Comment:
Why is this getting converted to a String when just below we're keeping it a Long?